lib/src/rover/orchestrator.dart

```diff
@@ -61,13 +63,13 @@ class RoverOrchestrator extends OrchestratorInterface with ValueReporter {
         collection.logger.debug(step.toString());
       }
       currentState = AutonomyState.DRIVING;
-      var count = 0;
+      // var count = 0;
```

Comment:
This was good because it catches small drifts in the GPS and IMU, allowing us to be more lenient about errors in everything. Maybe let's keep it and try removing it later when we do more physical tests

Reply:
Maybe instead of replanning after a set amount of steps, have it check after each step is completed if it has drifted too far and replan?

Reply:
Planning is very cheap -- not noticeable during operation. If there is no drift, the new plan == the old plan. So this is pretty much just as efficient as that, without the extra logic

Reply:
Doesn't replanning stop the rover suddenly? I'd rather not replan if it will cause a visible hiccup

Reply:
Not if we move `faceNorth` outside the while loop, Then there is no stop command and the re-planning will happen while the rover is still executing/finishing the last command. Technically there is a very slight delay but we're still unsure exactly what accuracy we will end up having. Keep in mind that we still don't have RTK so our GPS will have significant error.